ccloud-monitoring/docs/consumer-scenarios/increasing-consumer-lag.rst

```diff
@@ -0,0 +1,168 @@
+
+A large consumer lag, or a quickly growing lag, indicates that the consumer is not able to keep up with
+the volume of messages on a topic.
+
+This scenario will look at |ccloud| metrics from the Metrics API, `kafka-lag-exporter <https://github.com/lightbend/kafka-lag-exporter>`__ metrics, and
+client metrics from the client application’s MBean object ``kafka.consumer:type=consumer-fetch-manager-metrics,client-id=<client_id>``.
+
+Introduce failure scenario
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. By default 2 consumers and 1 producer are running. Adjust that to 1 consumer and 5 producers in order to show an increase in consumer lag.
+   The container scaling can be done with the command below:
+
+   .. code-block:: bash
+
+      docker-compose up -d --scale consumer=1 --scale producer=5
+
+   Which will produce the following output:
+
+   .. code-block:: bash
+
+      ccloud-exporter is up-to-date
+      kafka-lag-exporter is up-to-date
+      node-exporter is up-to-date
+      grafana is up-to-date
+      prometheus is up-to-date
+      Stopping and removing ccloud-monitoring_consumer_2 ... done
+      Starting ccloud-monitoring_producer_1              ... done
+      Creating ccloud-monitoring_producer_2              ... done
+      Creating ccloud-monitoring_producer_3              ... done
+      Creating ccloud-monitoring_producer_4              ... done
+      Creating ccloud-monitoring_producer_5              ... done
+      Starting ccloud-monitoring_consumer_1              ... done
+
+Diagnose the problem
+^^^^^^^^^^^^^^^^^^^^
+
+#. Open `Grafana <localhost:3000>`__ and login with the username ``admin`` and password ``password``.
+
+#. Navigate to the ``Consumer Client Metrics`` dashboard. Wait 2 minutes and then observe:
+
+   - ``Rebalance rate`` (``rebalance-rate-per-hour``) has a bump, indicating that the consumer group ``demo-cloud-monitoring-1`` underwent a rebalance, which is to be expected when a consumer leaves the group.
+
+   |Consumer Rebalance Bump|
+
+   - An upward trend in ``Consumer group lag in records``.  ``Consumer group lag in seconds`` will have a less dramatic increase.
+     Both indicating that the producer is creating more messages than the consumer can fetch in a timely manner.
+     These metrics are derived from the ``kafka-lag-exporter`` container.
+     `kafka-lag-exporter <https://github.com/lightbend/kafka-lag-exporter>`__ is a scala open source project that collects data about consumer groups and presents them in a Prometheus scrapable format.
+
+   |Consumer Lag|
+
+   - An increase in ``Fetch request rate`` (``fetch-total``) and ``Fetch size avg`` (``fetch-size-avg``) in the ``Consumer Fetch Metrics`` tab, indicating the consumer is fetching more often and larger batches.
+
+   |Consumer Fetch Increase|
+
+   - All of the graphs in the ``Throughput`` are indicating the consumer is processing more bytes/records.
```

Comment:
For this point and the previous point, while they are true, it doesn't necessarily mean that the consumer lag would necessarily increase (especially if the consumer is tuned/resources properly).  Would it make sense to make a note of this?

Reply:
I added a note. 
![Screen Shot 2021-02-25 at 10 58 35 AM](https://user-images.githubusercontent.com/12937975/109196992-ab9e3900-7759-11eb-89fe-abc10ad489ae.png)